aten/src/ATen/function_wrapper.py

```diff
@@ -588,6 +600,7 @@ def process_native(option, output_options):
         # method-only things are prefixed with m_ in Type so that
         # another function-only variant can exist without the name colliding
         option['method_prefix'] = 'm_' if is_method and not is_function else ''
+        option['method_prefix_derived'] = option['method_prefix']
```

Comment:
aside: The original need for 'm_' was because there were some function-only things named similarly to method-only things that were actually not the same declaration. I wonder if they still exist after the API changes Sam did. If not, we might be able to drop the 'm_' prefix.

Reply:
This worked -- it's in the latest commits.